public/index.html

```diff
@@ -0,0 +1,375 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+  <meta name="description" content="">
+  <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1,user-scalable=no">
+  <title>Ballers: Become A Landlord</title>
+  <link rel="apple-touch-icon" sizes="180x180" href="./apple-touch-icon.png">
+  <link rel="icon" type="image/png" sizes="32x32" href="./favicon-32x32.png">
+  <link rel="icon" type="image/png" sizes="16x16" href="./favicon-16x16.png">
+  <link rel="manifest" href="./site.webmanifest">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+  <link rel="stylesheet" type="text/css" href="./assets/slick/slick.css"/>
+  <link rel="stylesheet" type="text/css" href="./assets/slick/slick-theme.css"/>
+  <link rel="stylesheet" type="text/css" href="assets/css/ballers.css">
+
+</head>
+
+<body>
+  <!-- Holding -->
+  <section class="holding">
+
+    <!-- Header -->
+    <header>
+      <nav class="navbar navbar-expand-lg">
+        <a class="navbar-brand" href="#"><img class="ballers-logo" src="./assets/img/ballers-logo.png" alt=""></a>
```

Comment:
The link here should resolve to the home page. We also need to include an `alt` text for the image. Maybe 'Ballers Logo' 🤷‍♂